dev-lang/perl/perl-5.40.0.recipe

```diff
@@ -98,8 +98,12 @@ BUILD()
 		-Dprivlib=$prefix/lib/perl5/$portVersion \
 		-Dsiteprefix=$prefix/non-packaged \
 		-Dsitelib=$prefix/non-packaged/lib/perl5/site_perl/$perlShortVersion \
+		-Dsiteman1dir=$prefix/non-packaged/$relativeManDir/man1 \
+		-Dsiteman3dir=$prefix/non-packaged/$relativeManDir/man3 \
```

Comment:
How does this worked in a recipe? eg install system wide? From what I see here is user install related, but I could be wrong.

Reply:
Recipes don't get installed into the "site" directories, but into the "vendor" directories instead (either via [`HAIKU_USE_VENDOR_DIRECTORIES` in haikuporter](https://github.com/haikuports/haikuporter/blob/f666118fd401007c9323d45aeb6838e595a32646/HaikuPorter/Port.py#L1548) or manually, e.g. in [sdl_perl](https://github.com/haikuports/haikuports/blob/a068cc26b94a6fc7bdf7f3d2c6dcc9995d9a7f76/dev-perl/sdl_perl/sdl_perl-2.548.recipe#L77)).

The "site" directories are for manually installed ones, e.g. via cpan.

Reply:
Also, these paths use `$prefix`, so they are install-location agnostic and work for system and user installs.